### PR TITLE
Add FunctionName and FileName to the LogEntries sent to cloud.

### PR DESCRIPTION
**Pull request by kalyan-nidumolu**
Added Function Name and FileName to the LogEntry. THis allows these fields to be used for filtering the logs in Kibana

Testing Done
===========
Upgraded to the new image - added debugs and verified that the fields are getting filled. These new fields do not show up in kibana till the Proto files are updated in cloud.

### CODE REVIEW

File: pkg/pillar/cmd/logmanager/logmanager.go [hunk 1]
```diff
@@ -611,6 +614,7 @@ func HandleLogEvent(event logEntry, reportLogs *logs.LogBundle, counter int) {
 	logDetails.Timestamp, _ = ptypes.TimestampProto(event.timestamp)
 	logDetails.Source = event.source
 	logDetails.Iid = event.iid
+	logDetails.Iid = event.iid
```
Comment (eriknordmark) [inline pkg/pillar/cmd/logmanager/logmanager.go]: Duplicate line

**Comment by kalyan-nidumolu** [comment]
> I don't see any code which sets the new fields.
> Note that they are already included in the content in the API because they are in the output which logrus is producing in the log files we get from the go agents.
> Thus you need to extract the filename and function name from that json to fill in the new fields in the API.

Hi Erik,

the usage is in this line in logmanager.go:
  logChan <- logEntry{source: r.source,

File: pkg/pillar/cmd/logmanager/logmanager.go (hunk 1 above)
Comment (kalyan-nidumolu) [reply pkg/pillar/cmd/logmanager/logmanager.go]: Ah.. good catch.. Typo..will fix it.

**Comment by kalyan-nidumolu** [comment]
> Note that they are already included in the content in the API because they are in the output which logrus is producing in the log files we get from the go agents.

Good catch.. I thought I addressed it.. But missed the final hookup. Fixed it.

**Pull request merged**